IntegrationTestProject/ProjectUnderTest/TestQualityProfile.xml

```diff
@@ -0,0 +1 @@
+<?xml version='1.0' encoding='UTF-8'?><profile><name>ProfileForTest</name><language>cs</language><rules><rule><repositoryKey>fxcop</repositoryKey><key>DoNotRaiseReservedExceptionTypes</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>fxcop</repositoryKey><key>DoNotPassLiteralsAsLocalizedParameters</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>csharpsquid</repositoryKey><key>S2228</key><priority>CRITICAL</priority><parameters/></rule><rule><repositoryKey>csharpsquid</repositoryKey><key>S1134</key><priority>MAJOR</priority><parameters/></rule></rules></profile>
```

Comment:
This is a quality profile that I created and exported using the SonarQube dashboard. The quality profile is then restored on each SonarQube server. Note that managing quality profiles in SQ 4.* is not trivial because of a missing API. This approach was discussed with Dinesh and is also used by the Orchestrator (i.e. the Java equivalent of these scripts)